Bottom: 6c4fbd8f59082c52e8d6cc64b3e16c90761f0dcf
Top:    3b61dc0477bca30367d7c0c673f10f6283f6c098
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-06-11 23:09:55 +0100

Refresh of update-netclass-clearances

---

diff --git a/src/netclass.c b/src/netclass.c
index d13d3f0..54c6d6e 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
   };
